src/Firestore/Encode.elm

```diff
@@ -113,8 +113,8 @@ string value =
 
 
 {-| -}
-list : List a -> (a -> Field) -> Field
-list value valueEncoder =
+list : (a -> Field) -> List a -> Field
```

Comment:
Nice catch! Yes, this interface should have followed `Json.Encode.list`, and now it's done :+1: